sonata.core

```diff
@@ -49,7 +49,7 @@ parameters:
   SRAMInitFile:
     datatype: str
     description: SRAM initialization file in vmem hex format
-    default: "../../../../../sw/cheri/cheri_sanity/cpu0_irom.vmem"
+    default: "../../../../../build_cheri/tests/cheri_sanity.vmem"
```

Comment:
The convention we use for ibex-demo-system and legacy SW is probably to put it in `sw/cheri/build`